src/components/user/DeleteAccountPage.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import trashIcon from '../../assets/trash-icon.png'; // Path to trash icon asset in src/assets
+import logoWithCircleBorder from '../../assets/logo_with_circle_border-removebg.png'; // Path to novelology logo in assets
+
+// Page Container Style
+const pageContainerStyle = () => ({
+    backgroundColor: 'rgba(91, 46, 72, 1)', // Set background color here
+    width: '100%',
+    minHeight: '100vh', // Set minimum height to cover the entire viewport
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    overflow: 'hidden',
+});
+
+// Main Content Area Style
+const mainContentStyle = () => ({
+    backgroundColor: 'rgba(91, 46, 72, 1)',
+    width: '100%',
+    minHeight: '60vh',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-around',
+    alignItems: 'center',
+    position: 'relative',
+    overflow: 'hidden',
+    padding: '20px',
+    boxSizing: 'border-box',
+});
+
+// Group of Texts Style
+const textGroupStyle = () => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '100%',
+});
+
+// Logo Image Style
+const logoImageStyle = () => ({
+    marginTop: '10px',
+    width: '180px',
+    height: 'auto',
+    marginBottom: '50px',
+});
+
+// Trash Icon Style
+const trashIconStyle = () => ({
```

Comment:
The filter to set the color of the icon would be unnecessary if we used FontAwesome Icons instead